src/componenets/Calculator/Calculator.js

```diff
@@ -0,0 +1,31 @@
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator-grid">
+      <div className="input-output">444</div>
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="btncolor">÷</button>
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="btncolor">x</button>
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button" className="btncolor">-</button>
+      <button type="button">1</button>
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button type="button" className="btncolor">+</button>
+      <button type="button" className="span-two">0</button>
+      <button type="button">.</button>
+      <button type="button" className="btncolor">=</button>
+    </div>
+
+  );
+}
```

Comment:
- [ ] Great job so far. :muscle: You did a great job following the project requirements. However, I noticed that you missed one.
-  According to the [Project Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) the `Calculator component` should have at least one child component.  That component can handle certain specific parts of the `Calculator component`. :wink: 
- This way, you can create simpler and more concise code that is easier to understand and maintain. :100: